test/cpp/api/rnn.cpp

```diff
@@ -110,15 +110,15 @@ TEST_CASE("rnn") {
     LSTM model(2, 2);
     for (auto& v : model->parameters()) {
       float size = v->numel();
-      auto p = static_cast<float*>(v->data().storage()->pImpl()->data());
```

Comment:
why do we allow -> on storages?  There can't be that many useful methods, right?  Just move them to Storage directly?  Because we don't have the issue that we have with Tensor where we need a ton of methods here (i.e. TensorMethods.h).

Reply:
Per offline conversation, "Just because I'm lazy." We can add the methods on them pretty easy later.